packages/share_plus/share_plus/lib/share_plus.dart

```diff
@@ -18,6 +18,19 @@ export 'src/share_plus_windows.dart'
 class Share {
   static SharePlatform get _platform => SharePlatform.instance;
 
+  /// Summons the platform's share sheet to share uri.
+  ///
+  /// Wraps the platform's native share dialog. Can share a URL.
+  /// It uses the `ACTION_SEND` Intent on Android and `UIActivityViewController`
+  /// on iOS. [shareUri] will trigger the iOS system to fetch the html page
+  /// (if available), and the website icon will be extracted and displayed on
+  /// the iOS share sheet.
+  static Future<void> shareUri(
+    Uri uri,
+  ) async {
+    return _platform.shareUri(uri);
```

Comment:
This is why one test is failing

```
    error - lib/share_plus.dart:31:22 - The method 'shareUri' isn't defined for the type 'SharePlatform'. Try correcting the name to the name of an existing method, or defining a method named 'shareUri'. - undefined_method
```

Reply:
The check take share_plus_platform_interface: ^3.2.1 from pub.dev rather than your changes here.
Do we first need to deploy the PI, before the tests are passing, or could Melos fix it, @vbuberen?

Reply:
Sorry for a late response. Melos should take care of picking up changes in platform interface when they are done locally.